src/table/components/TableWrapper.jsx

```diff
@@ -152,13 +154,23 @@ export default function TableWrapper(props) {
         data-testid="table-container"
       >
         <Table stickyHeader aria-label={tableAriaLabel}>
-          <TableHeadWrapper {...props} />
+          <TableHead>
+            <TableHeadWrapper
+              {...props}
+              setFocusedCellCoord={setFocusedCellCoord}
+              focusedCellCoord={focusedCellCoord}
+            />
+            {totalsPosition === 'top' && <TableTotals {...props} setFocusedCellCoord={setFocusedCellCoord} />}
```

Comment:
setFocusedCellCoord should not be passed as a props since it exists on the `useContextSelector`. use That instead inside the totals function